cuegui/cuegui/RequestCoresDialog.py

```diff
@@ -0,0 +1,362 @@
+        self.__email_body_user_date_info.moveCursor(QtGui.QTextCursor.Start)
+        self.__email_body_user_date_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_date_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_date_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_date_info.moveCursor(QtGui.QTextCursor.Down)
+
+        self.__email_body_user_extra_info.setFocus(QtCore.Qt.OtherFocusReason)
+        self.__email_body_user_extra_info.moveCursor(QtGui.QTextCursor.Start)
+        self.__email_body_user_extra_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_extra_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_extra_info.moveCursor(QtGui.QTextCursor.Down)
+        self.__email_body_user_extra_info.moveCursor(QtGui.QTextCursor.Down)
+
+    def build_html_body(self):
+        """Format html body for multipart email."""
+        html_body_prefix = self.EMAIL_REQUEST_CORES_USER_HTML_BODY_PREFIX.format(
+            self.__job.name(),
+            self.__job.group(),
+            self.EMAIL_REQUEST_CORES_BODY_LAYERS_PREFIX,
+        )
+        layers_data = ""
+        for layer in self.__job.getLayers():
+            layers_data += self.EMAIL_REQUEST_CORES_HTML_BODY_TABLE_LAYERS.format(
+                layer.name(), layer.minMemory(), layer.data.min_cores
+            )
+
+        date_time_info = self.__email_body_user_date_info.toPlainText()
+        extra_info = self.__email_body_user_extra_info.toPlainText()
+        html_body = self.EMAIL_REQUEST_CORES_HTML_BODY.format(
+            html_body_prefix,
+            layers_data,
+            self.EMAIL_REQUEST_CORES_USER_DATE_PREFIX,
+            date_time_info,
+            self.EMAIL_REQUEST_CORES_USER_ADDITIONAL_PREFIX,
+            extra_info,
+        )
+
+        return html_body
+
+    def email_from(self):
+        """Gets the email sender."""
+        return "%s" % self.__email_from.text()
+
+    def email_to(self):
+        """Gets the email recipient."""
+        return "%s" % self.__email_to.text()
+
+    def email_cc(self):
+        """Gets the email CC field."""
+        return "%s" % self.__email_cc.text()
+
+    def email_bcc(self):
+        """Gets the email BCC field."""
+        return "%s" % self.__email_bcc.text()
+
+    def email_subject(self):
+        """Gets the email subject."""
+        return "%s" % self.__email_subject.text()
+
+    def email_body(self):
+        """Get the email body text."""
+        return "%s\n\n%s\n%s" % (
+            self.__email_body.toPlainText(),
+            self.__email_body_user_date_info.toPlainText(),
+            self.__email_body_user_extra_info.toPlainText(),
+        )
+
+    def appendToDefaultBody(self, txt):
+        """Add information to body the email."""
+        self.__email_body.append(txt)
+        font = self.__email_body.document().defaultFont()
```

Comment:
In Python, the snake_case is a variable naming convention:

Recommended change to:

```python
font_metrics = QtGui.QFontMetrics(font)
text_size = font_metrics.size(0, txt)
text_height = text_size.height() + 30
text_width = text_size.width()
self.__email_body.resize(text_width, text_height)
```

Reply:
Python [pep8 style guide](https://peps.python.org/pep-0008/#constants) states that "constants" are all capital letters